### DISCUSSION TITLE
Изменение строки, в том случае когда остальные строки true/false

### DISCUSSION THREAD

**Discussion by Murbulat1** [totumonline/totum-ru-issues-and-discussions#429] [Как это сделать?] [answered]
Всем привет, подскажите пожалуйста как реализовать.
Есть два столбца, "название"(как строка) и "готовность"(чекбокс). В футоре есть столбец "f_ready_2" , там есть два варианта : один, два.  Как сделать, если в столбце "готовность" все строки true, то в столбце f_ready2 - два. Если же, в столбце "готовность" все строки false, то в столбце f_ready2 - один. 

Пытался сделать через if 

```
 `= : if(condition: $list = true; then: $true ; else: $false)
	list: listCheck(list: $listing)
		listing: selectList(table: $#ntn; field: 'itogo')
		true: str`'one'`
		false: str`'twoo'`
```

но, там когда 1 строка меняет false на true сразу переключается. А мне нужно когда все строки либо true либо false. 

![image](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/92213625/feb17767-1ecb-4bc0-9a64-3c469110b3d8) ![image](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/92213625/bd5406c5-94b2-4884-9556-277bd8dc2562)

**Comment by erpic-ru** [comment, marked as answer]
Делайте просто 2 селекта по строкам и ищите false, если не находит - значит все true, аналогично для true.

**Reply by Murbulat1** [reply]
Спасибо, через селект сработало!
